homeassistant/components/epson/config_flow.py

```diff
@@ -28,30 +28,25 @@ async def async_step_import(self, import_config):
         for entry in self._async_current_entries(include_ignore=True):
             if import_config[CONF_HOST] == entry.data[CONF_HOST]:
                 return self.async_abort(reason="already_configured")
-        try:
-            projector = await validate_projector(
-                self.hass,
-                import_config[CONF_HOST],
-                check_power=True,
-                check_powered_on=False,
-            )
-        except CannotConnect:
-            _LOGGER.warning("Cannot connect to projector")
-            return self.async_abort(reason="cannot_connect")
-        else:
-            serial_no = await projector.get_serial_number()
-            await self.async_set_unique_id(serial_no)
-            self._abort_if_unique_id_configured()
-            return self.async_create_entry(
-                title=import_config.pop(CONF_NAME), data=import_config
-            )
+        return await self.combine_steps(
```

Comment:
This will potentially go directly to the user form. We shouldn't do that. Each form is private to its step method. If we need to change step, we should await the corresponding step method.

Generally for config flow helpers I recommend to not call the form but raise a custom exception that step methods can catch and keep the form call in the step method.